src/monitoring_service/blockchain.py

```diff
@@ -15,13 +15,17 @@
     ReceiveChannelSettledEvent,
     ReceiveNonClosingBalanceProofUpdatedEvent,
     UpdatedHeadBlockEvent,
+    ReceiveMonitoringNewBalanceProofEvent,
+    ReceiveMonitorinRewardClaimedEvent,
```

Comment:
There a "g" missing in "ReceiveMonitorin"

Reply:
Nice catch.